skynet/env.py

```diff
@@ -87,7 +88,6 @@ def tobool(val: str | None):
 job_timeout = int(os.environ.get('JOB_TIMEOUT', 60 * 5))  # 5 minutes default
 
 # summaries
-summary_default_hint_type = os.environ.get('SUMMARY_DEFAULT_HINT_TYPE', 'text')
```

Comment:
Why are we removing this? Is it related to the vllm change?

Reply:
not removed, but the default value type (string) does not match the declared pydantic type, which was an Enum, so I moved it there to use the local declaration of that Enum